static/app/views/settings/components/dataScrubbing/organizationRules.tsx

```diff
@@ -44,9 +60,18 @@ class OrganizationRules extends Component<Props, State> {
     }));
   };
 
+  loadRules() {
+    try {
+      this.setState({
+        rules: convertRelayPiiConfig(this.props.organization.relayPiiConfig),
+      });
+    } catch {
+      addErrorMessage(t('Unable to load organization rules'));
```

Comment:
You had an error message with `Unable to load data scrubbing rules` above -- should this be consistent?